pom.xml

```diff
@@ -42,7 +47,17 @@
 			<groupId>ai.aitia</groupId>
 			<artifactId>arrowhead-application-library-java-spring</artifactId>
 			<version>4.4.0.0</version>
-		</dependency>		
+		</dependency>
+		<dependency>
+			<groupId>org.eclipse.paho</groupId>
+			<artifactId>org.eclipse.paho.client.mqttv3</artifactId>
```

Comment:
I think this should belong to the `arrowhead-application-library-java-spring` an have similar method(s) in `ArrowheadService` like the WebSocket support has.